From a573ed99aaf60900d5f6729aa470ceeb64a6ccc8 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Tue, 30 May 2023 10:45:29 +0530
Subject: [PATCH] Update security_patch_level string

Update security_patch_level as the ASB
pacthes are integrated

Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 419ff1aadc..fbbe777754 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -103,7 +103,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2023-05-05
+    PLATFORM_SECURITY_PATCH := 2024-02-01
 endif
 
 include $(BUILD_SYSTEM)/version_util.mk
-- 
2.17.1

